alby/alby_oauth_service.go

```diff
@@ -396,6 +396,11 @@ func (svc *albyOAuthService) GetAuthUrl() string {
 }
 
 func (svc *albyOAuthService) LinkAccount(ctx context.Context, lnClient lnclient.LNClient, budget uint64, renewal string) error {
+	appName := "getalby.com"
```

Comment:
what happens if I create one and call it like that?

Reply:
There will be :dragon: